actions/drain_node_handler.go

```diff
@@ -139,8 +163,18 @@ func (h *drainNodeHandler) evictNodePods(ctx context.Context, log logrus.FieldLo
 	}
 
 	log.Infof("evicting %d pods", len(pods))
+	if len(pods) == 0 {
+		return nil
+	}
+	groupVersion, err := drain.CheckEvictionSupport(h.clientset)
```

Comment:
If eviction is not supported, but force drain is selected, should we care?

Reply:
This is not about checking if eviction is supported but with  version is supported. 
> Note: policy/v1 Eviction is available in v1.22+. Use policy/v1beta1 with prior releases.

I don't think that we have customer with K8s cluster that doesn't support eviciton.